demos/blockfactory/utils/index.html

```diff
@@ -0,0 +1,14 @@
+<!--
+  Copyright 2017 Juan Carlos Orozco Arena
+  Apache License Version 2.0
+-->
+<html>
+  <head>
+    <script src="./block_factory_curated_xml.js"></script>
+  </head>
+  <body>
+    <h1>Convert xml to js</h1>
+    <div>Generates an xml building library.</div>

```

Comment:
Please elaborate this comment. What is the role of the output XML library? What is its relation to `block_factory_xml_build_lib.js` (the output, right?) and how does the rest of the system use it?

Similarly, can we give this file a more descriptive filename?